### PR TITLE
fix php call causing breakage as early as redcap v10.5.1

### PR DESCRIPTION
**Pull request by ChemiKyle**
hardcode arm arg in enableDataSearchAutocomplete call

~TODO: README update~

### CODE REVIEW

**Comment by mbentz-uf** [comment]
I forgot, where can I find a sample project to test this?

**Comment by ChemiKyle** [comment]
You'll want to use the Cases and Person Search projects from redcap-phone, use Person Search as the source project and Cases as the target with the mapping files under `examples/*.json`. All the module set up will be in the target project only, though you'll need to provide some seed data to Person Search (manually entering a few is fine but you can script it if you like by pulling from the STP DB's `person` table and `sample_frac`'ing).  
We can do a video call if you need some help setting up.

Note that the bug this fixes occurs when trying to pipe data, searching still works.

**Comment by mbentz-uf** [comment]
Was multi-arm supported in previous versions of REDCap and break with a newer release? Maybe the pr can renamed to "_Remove multi-arm support_" instead.

**Comment by ChemiKyle** [comment]
No, arms were never supported, not mentioning that was an oversight.

**Comment by mbentz-uf** [comment]
I understand, this is a fix to get the module up and running. Multi-arm support can either be added or removed entirely later via:
https://github.com/ctsit/search_and_populate_data_from_another_project/blob/37613f41893742102248199ddf1a9f6e8d22b79b/js/custom_data_search.js#L22

File: js/custom_data_search.js [hunk 1]
```diff
@@ -59,13 +59,13 @@ $( document ).ready( function() {
     }
     $(function(){
         // Enable searching via auto complete
-        enableDataSearchAutocomplete($('#field_select option:first').val(),'<?php echo getArm() ?>');
+        enableDataSearchAutocomplete($('#field_select option:first').val(),'1');
```
Comment (mbentz-uf) [inline js/custom_data_search.js:62]: Might need to revisit this to add or completely remove multi-arm support.

**Review by mbentz-uf — APPROVED** [review]

File: js/custom_data_search.js (hunk 1 above)
Comment (ChemiKyle) [reply js/custom_data_search.js:62]: Yeah, I wanted to put this off until we do a UI redesign; I envision that if the target project form had multiple arms you'd get a dropdown to select one.  
That PHP echo would be setting the arm from the _source_ project if it were working.

**Pull request merged**